### DISCUSSION TITLE
Helm-chart - Activating Multi-tenancy

### DISCUSSION THREAD

**Discussion by kevinremoue** [grafana/mimir#2239] [Help and support] [answered]
Hello,

I am exploring Mimir and I saw in the helm chart values the following:

`    {{- if not .Values.enterprise.enabled -}}
    multitenancy_enabled: false
    {{- end }}
`
I want to achieve multi-tenancy using X-Scope-OrgID headers configured in my different prom/vector agents, is this only available in the enterprise edition ? 
(Im talking about [this](https://grafana.com/docs/mimir/latest/operators-guide/securing/authentication-and-authorization/#without-an-authenticating-reverse-proxy))

**Comment by replay** [comment, marked as answer]
Open source Mimir also supports multi-tenancy via the X-Scope-OrgID header, it is enabled by default.

**Reply by kevinremoue** [reply]
Okay but is it related to this config ? `multitenancy_enabled`
If yes, then why does it depends on enterprise license...

**Reply by Mojito2110** [reply]
Hi,

In the next version 2.2.0 (tested with 2.2.0-rc0) the template changed : the multi-tenancy for mimir is enabled by default without mimir-enterprise enabled

**Reply by kevinremoue** [reply]
I just tried again using 2.2.0-weekly.191, it is working now. 
Thanks!
